Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3255,6 +3262,12 @@ public boolean onSearchRequested() {
             return callback_.onSearchRequested();
         }
 
+        @TargetApi(Build.VERSION_CODES.M)
+        @Override
+        public boolean onSearchRequested(SearchEvent searchEvent) {
```

Comment:
SDK is built with Android 23 , which has 2 new methods added for Branch Window Callback.